### PR TITLE
Change form strings

### PR DESCRIPTION
**Pull request by Salomari1987**

### CODE REVIEW

**Comment by ghost** [comment]
Thanks @Salomari1987. There's a couple of failed tests related to the change:

Could you please update the related tests to have just `Username`:

https://github.com/Edraak/edraak-platform/blob/efd4e68cc02928b10cf478d564b95d646141f4b4/openedx/core/djangoapps/user_api/tests/test_views.py#L1137

and 

https://github.com/Edraak/edraak-platform/blob/efd4e68cc02928b10cf478d564b95d646141f4b4/openedx/core/djangoapps/user_api/tests/test_views.py#L1340


I've attached a screenshot from Jenkins in case you don't have access yet:

<kbd>image</kbd>

File: conf/locale/en/LC_MESSAGES/djangojs.po [hunk 1]
```diff
@@ -3276,19 +3276,20 @@ msgstr ""
 
 #: common/static/bundles/StudentAccountDeletion.js
 #: common/static/bundles/StudentAccountDeletionInitializer.js
-#: lms/static/js/student_account/components/StudentAccountDeletion.jsx
 msgid ""
 "Please note: Deletion of your account and personal data is permanent and "
-"cannot be undone. Edraak will not be able to recover your account or the "
-"data that is deleted."
+"cannot be undone. EdX will not be able to recover your account or the data "
```
Comment (Fahminajjar) [inline conf/locale/en/LC_MESSAGES/djangojs.po:3281]: Why EdX instead of Edraak?

File: conf/locale/en/LC_MESSAGES/djangojs.po [hunk 2]
```diff
@@ -3276,19 +3276,20 @@ msgstr ""
...
+"that is deleted."
 msgstr ""
 
 #: common/static/bundles/StudentAccountDeletion.js
 #: common/static/bundles/StudentAccountDeletionInitializer.js
-#: lms/static/js/student_account/components/StudentAccountDeletion.jsx
 msgid ""
-"Once your account is deleted, you cannot use it to take courses on the "
-"Edraak app, edraak.org, or any other site hosted by Edraak."
+"Once your account is deleted, you cannot use it to take courses on the edX "
```
Comment (Fahminajjar) [inline conf/locale/en/LC_MESSAGES/djangojs.po:3288]: Same here.

File: conf/locale/en/LC_MESSAGES/djangojs.po [hunk 3]
```diff
@@ -3332,19 +3333,20 @@ msgstr ""
 
 #: common/static/bundles/StudentAccountDeletion.js
 #: common/static/bundles/StudentAccountDeletionInitializer.js
-#: lms/static/js/student_account/components/StudentAccountDeletionModal.jsx
 msgid ""
 "You have selected “Delete my account.” Deletion of your account and personal"
-" data is permanent and cannot be undone. Edraak will not be able to recover "
+" data is permanent and cannot be undone. EdX will not be able to recover "
```
Comment (Fahminajjar) [inline conf/locale/en/LC_MESSAGES/djangojs.po:3338]: Same here.

File: conf/locale/en/LC_MESSAGES/djangojs.po [hunk 4]
```diff
@@ -3332,19 +3333,20 @@ msgstr ""
...
 "your account or the data that is deleted."
 msgstr ""
 
 #: common/static/bundles/StudentAccountDeletion.js
 #: common/static/bundles/StudentAccountDeletionInitializer.js
-#: lms/static/js/student_account/components/StudentAccountDeletionModal.jsx
 msgid ""
 "If you proceed, you will be unable to use this account to take courses on "
-"the Edraak app, edraak.org, or any other site hosted by Edraak."
+"the edX app, edx.org, or any other site hosted by edX. This includes access "
```
Comment (Fahminajjar) [inline conf/locale/en/LC_MESSAGES/djangojs.po:3346]: Same here.

**Review by Fahminajjar — CHANGES_REQUESTED** [review]

File: conf/locale/eo/LC_MESSAGES/djangojs.po [hunk 5]
```diff
@@ -3568,30 +3568,36 @@ msgstr "Wé’ré sörrý tö séé ýöü gö! Ⱡ'σяєм ιρѕυм ∂σł
 
 #: common/static/bundles/StudentAccountDeletion.js
 #: common/static/bundles/StudentAccountDeletionInitializer.js
-#: lms/static/js/student_account/components/StudentAccountDeletion.jsx
 msgid ""
 "Please note: Deletion of your account and personal data is permanent and "
-"cannot be undone. Edraak will not be able to recover your account or the "
-"data that is deleted."
+"cannot be undone. EdX will not be able to recover your account or the data "
```
Comment (Fahminajjar) [inline conf/locale/eo/LC_MESSAGES/djangojs.po:3573]: Same here.

File: conf/locale/en/LC_MESSAGES/djangojs.po [hunk 6]
```diff
@@ -5659,6 +5661,32 @@ msgstr ""
...
+"You have selected “Delete my account.” Deletion of your account and personal"
+" data is permanent and cannot be undone. Edraak will not be able to recover "
+"your account or the data that is deleted."
+msgstr ""
+
+#: lms/static/js/student_account/components/StudentAccountDeletionModal.jsx
+msgid ""
+"If you proceed, you will be unable to use this account to take courses on "
+"the Edraak app, edraak.org, or any other site hosted by Edraak."
+msgstr ""
```
Comment (Salomari1987) [inline conf/locale/en/LC_MESSAGES/djangojs.po:5688]: Check these, I think these should be fine

**Review by Fahminajjar — APPROVED** [review]

**Pull request merged**